packages/getMyPrescriptions/tests/statusUpdate.test.ts

```diff
@@ -48,6 +48,12 @@ describe("Unit tests for statusUpdate", () => {
     const result = buildStatusUpdateData(bundle)
     expect(result).toEqual([])
   })
+
+  test("when a bundle is passed-in, all items have a status of either Prescriber Approved or Cancelled", async () => {
```

Comment:
The description of this test doesn't make its intention clear. Plus, it's doing what the earlier test is doing.

I'd suggest something a bit more explicit, like using the original test data (without your change), but adding extensions to one of the prescriptions. That would then make result.length equal 1.

Reply:
The test that excludes prescriptions where all items have a status of either 'Prescriber Approved' or 'Cancelled' has been added.